tests/customer-resource-deselection-test.js

```diff
@@ -137,7 +137,7 @@ describeApplication('CustomerResourceShow Deselection', () => {
 
             it('is not hidden', () => {
               expect(resource.visibilityData.isHidden).to.equal(false);
-              expect(ResourcePage.isHidden).to.equal(false);
+              expect(ResourcePage.isHidden).to.equal(true);
```

Comment:
I think this should still be `expect(ResourcePage.isHidden).to.equal(false);` - the page helper function might need to be switched around.